LabGui.py

```diff
@@ -1512,6 +1532,19 @@ def tool_widgetmanager(self):
         global USERWIDGET_MANAGER
         USERWIDGET_MANAGER = UserWidgetManager.UserWidgetManager(self)
         USERWIDGET_MANAGER.show()
+
+    def connectInstrToolbar(self):
```

Comment:
connect_instr_toolbar is the recommended [style](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) for function in python camel case is used in javascript for functions and variable names

Reply:
old habits die hard I guess haha